pkg/cloudprovider/providers/openstack/openstack_test.go

```diff
@@ -60,57 +60,57 @@ func WaitForVolumeStatus(t *testing.T, os *OpenStack, volumeName string, status
 	}
 }
 
-func TestReadConfig(t *testing.T) {
-	_, err := readConfig(nil)
-	if err == nil {
-		t.Errorf("Should fail when no config is provided: %s", err)
-	}
-
-	cfg, err := readConfig(strings.NewReader(`
-[Global]
-auth-url = http://auth.url
-username = user
-[LoadBalancer]
-create-monitor = yes
-monitor-delay = 1m
-monitor-timeout = 30s
-monitor-max-retries = 3
-`))
-	if err != nil {
-		t.Fatalf("Should succeed when a valid config is provided: %s", err)
-	}
-	if cfg.Global.AuthUrl != "http://auth.url" {
-		t.Errorf("incorrect authurl: %s", cfg.Global.AuthUrl)
-	}
-
-	if !cfg.LoadBalancer.CreateMonitor {
-		t.Errorf("incorrect lb.createmonitor: %t", cfg.LoadBalancer.CreateMonitor)
-	}
-	if cfg.LoadBalancer.MonitorDelay.Duration != 1*time.Minute {
-		t.Errorf("incorrect lb.monitordelay: %s", cfg.LoadBalancer.MonitorDelay)
-	}
-	if cfg.LoadBalancer.MonitorTimeout.Duration != 30*time.Second {
-		t.Errorf("incorrect lb.monitortimeout: %s", cfg.LoadBalancer.MonitorTimeout)
-	}
-	if cfg.LoadBalancer.MonitorMaxRetries != 3 {
-		t.Errorf("incorrect lb.monitormaxretries: %d", cfg.LoadBalancer.MonitorMaxRetries)
-	}
-}
-
-func TestToAuthOptions(t *testing.T) {
-	cfg := Config{}
-	cfg.Global.Username = "user"
-	// etc.
-
-	ao := cfg.toAuthOptions()
-
-	if !ao.AllowReauth {
-		t.Errorf("Will need to be able to reauthenticate")
-	}
-	if ao.Username != cfg.Global.Username {
-		t.Errorf("Username %s != %s", ao.Username, cfg.Global.Username)
-	}
-}
+ func TestReadConfig(t *testing.T) {
```

Comment:
Why is this function indented by a space? (and `TestNewOpenStack`, `TestInstances` below)